client/src/main/java/com/arcbees/beeshop/client/application/product/ProductView.java

```diff
@@ -93,19 +92,28 @@
     @UiField(provided = true)
     BrandPicker brandPicker;
     @UiField
-    LIElement smallAnchor;
+    LIElement smallLI;
     @UiField
-    LIElement mediumAnchor;
+    LIElement mediumLI;
     @UiField
-    LIElement largeAnchor;
+    LIElement largeLI;
     @UiField
-    LIElement xlargeAnchor;
+    LIElement xlargeLI;
+    @UiField
+    AnchorElement smallAnchor;
```

Comment:
Sometimes it's hard to name things, in the end they're list elements and anchors... Not much more meaning :-/